content/docs/users/_index.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Users"
+weight = 100
+template = "docs/index.html"
+[extra]
+emoji = "🧑"
+tile = "I want to start chatting using Matrix"
+updated = "2022-10-18T16:00:00Z"
++++
+
+## Matrix for Instant Messaging
+
+Matrix works a little like email, but instantaneous and secure:
+- You need to register an account at a provider
```

Comment:
Can we do an empty line between paragraph and list? I know most markdown parsers don't require it but the original markdown spec requires it for the list to actually be interpreted as a list.